amaznot/app/Http/Controllers/OrderController.php

```diff
@@ -22,16 +22,18 @@ public function index(Request $request)
             return $redirect;
         }
 
-        $orders = Order::all();
+        $orders = Order::where('user_id', Auth::user()->id)->orderBy('user_id','Desc')->get();
+        $orders = Order::paginate(10);
```

Comment:
You are making 2 queries here and the second one overwrites the data from the first one. What ends up happening is $orders gets written with data from the first query with the user chosen, then that gets overwritten by the second query which paginates from the orders table without regards to users. This should work instead:

$orders = Order::select('id', 'user_id', 'total', 'credit_card', 'created_at')
                        ->where('user_id', Auth::user()->id)
                        ->orderBy('user_id','Desc')
                        ->paginate(10);

Reply:
Oh, ok i'll make the change this for the $order and the $orderitems